### DISCUSSION TITLE
how to use ijavascipt kernel

### DISCUSSION THREAD

**Discussion by MrYZhou** [microsoft/vscode-jupyter#11047] [Questions and Answers] [answered]
ijsinstall fail:

error info:

$ bash ijsinstall
Error running `jupyter --version`
Error: Command failed: jupyter --version
'jupyter' is not recognized as an internal or external command,
operable program or batch file.

Error running `ipython --version`
Error: Command failed: ipython --version
'ipython' is not recognized as an internal or external command,
operable program or batch file.

but ,in last step, i has execute pip install jupter

**Comment by rchiodo** [comment, marked as answer]
If `ijsinstall` requires jupyter, you could follow the directions here:
https://docs.jupyter.org/en/latest/install/notebook-classic.html

That presumes you have a working python environment to install into.

**Reply by MrYZhou** [reply]
thanks for your quikly response ,i have solved.
i forget this command"conda install nodejs"
